ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -57,6 +57,10 @@ onnx_model_scatterND
 onnx_model_gatherND_int32
 onnx_model_gatherND_float
 
+# Not supported ONNX op: Range
+onnx_model_range_positive_step
+onnx_model_range_negative_step
```

Comment:
How can we check this functionality if tests are disabled for IE plugins?

Are these tests validated by INTERPRET plugin?
If no, can we introduce new test?

Reply:
These tests run on Interpreter. 
Since there isn't any support for dynamic shapes in the IP plugins, we can check the Range op functionality only using Interpreter.